tests/agent_equipment/test_model.py

```
@@ -0,0 +1,338 @@
+            )
+
+        with self.assertRaises(ValueError):
+            InstalledImplementationManifest(
+                schema_version="agent-equipment-installed-implementation/v1",
+                runtime_identity="cpython:3.12.8",
+                runtime_executable_digest="sha256:" + "1" * 64,
+                files=(),
+                digest="invalid",
+            )
+
+        with self.assertRaises(ValueError):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)